src/dev/impl/DevToys/Views/Tools/Converters/NumberBaseConverter/NumberBaseConverterToolPage.xaml.cs

```diff
@@ -47,21 +47,22 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
                 DataContext = ViewModel;
             }
 
-            if (!string.IsNullOrWhiteSpace(parameters.ClipBoardContent))
-            {
-                string clipBoardContent = NumberBaseFormatter.RemoveFormatting(parameters.ClipBoardContent).ToString();
+            // TODO: This feature would be nice to find a way to keep it
+            //if (!string.IsNullOrWhiteSpace(parameters.ClipBoardContent))
+            //{
+            //    string clipBoardContent = NumberBaseFormatter.RemoveFormatting(parameters.ClipBoardContent).ToString();
 
-                if (NumberBaseHelper.IsValidBinary(clipBoardContent!))
-                {
-                    ViewModel.InputBaseNumber = NumberBaseFormat.Binary;
-                }
-                else if (NumberBaseHelper.IsValidHexadecimal(clipBoardContent!))
-                {
-                    ViewModel.InputBaseNumber = NumberBaseFormat.Hexadecimal;
-                }
+            //    if (NumberBaseHelper.IsValidBinary(clipBoardContent!))
+            //    {
+            //        ViewModel.InputBaseNumber = NumberBaseFormat.Binary;
+            //    }
+            //    else if (NumberBaseHelper.IsValidHexadecimal(clipBoardContent!))
+            //    {
+            //        ViewModel.InputBaseNumber = NumberBaseFormat.Hexadecimal;
+            //    }
 
-                ViewModel.InputValue = parameters.ClipBoardContent;
-            }
+            //    ViewModel.InputValue = parameters.ClipBoardContent;
+            //}
```

Comment:
It would be good to revert this changes and keep this feature. This could require big changes but if you give me any idea I will try to fix it.

Reply:
What's the challenge you're encounter here? Is the problem to determine whether it's an advanced mode or not?

Reply:
Yes, the tool opens through NumberBaseConverter main view but the functionalities are inside Basic/Advanced view models, who are not declared inside the main view. However, I just realized that I could get this done using messages, isn't it? Still remains the problem of selecting between basic or advanced view...

Reply:
Ah I see. You should be able to do this by:
1. Set `x:Name="AdvancedNumberBaseConverterControl"` on the advanced control in the main view
2. And then, something like:

```
ViewModel.AdvancedMode = true;
AdvancedNumberBaseConverterControl.ViewModel.Property = something;
```